Styles/style1.css

```diff
@@ -0,0 +1,23 @@
+.portfolio {
```

Comment:
I'm not sure as to the purpose of this file.  Why not put these rules in `style.css`